homeassistant/components/tado/sensor.py

```diff
@@ -125,6 +125,12 @@ def unit_of_measurement(self):
         if self.zone_variable == "ac":
             return None
 
+    @property
+    def device_class(self):
+        """Return the device class."""
+        if self.zone_variable == "humidity":
+            return self.zone_variable
```

Comment:
Please use one of the official device class constants.

https://github.com/home-assistant/core/blob/4e71be852af8b42a3f43c8da79df4bb956439a80/homeassistant/components/sensor/__init__.py#L37-L50